soak/helm/ack-soak-test/templates/job.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: {{ .Values.awsService }}-soak-test
+  namespace: {{ .Release.Namespace }}
```

Comment:
Hmm this is odd. This shouldn't be necessary for Helm 3 if setting `--namespace` in each of the commands.

Reply:
We are not setting --namespace in each of the command. I tried doing that but it was being ignored. There is a bug in helm repo https://github.com/helm/helm/issues/3553

That's how I found about `kubectl config --namespace` command and it worked.

Reply:
This is regarding `helm template`. I see we are using `helm install`, which should not have this bug?